examples/autoqasm/6_Customize_gate_calibrations.ipynb

```diff
@@ -0,0 +1,404 @@
+    "\n",
+    "print(pulse_sequence_xy_theta_q0_q1.to_ir())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d4d9641",
+   "metadata": {},
+   "source": [
+    "## Compose custom gate calibrations with AutoQASM\n",
+    "To customize a gate calibration, you need to \n",
+    "1. Find the definition of the gate.\n",
+    "2. Decide the qubits and angles you want to define the calibration.\n",
+    "3. Compose a pulse program that defines the calibration.\n",
+    "\n",
+    "As an example, let's define the gate calibration of `rx` on qubit `$1` and at angle `pi/2`. We first inspect the definition of the `rx` gate, which takes qubit parameter `target` and angle parameter `angle`. A gate calibration for the `rx` gate must fully specify the inputs of the gate, `target` and `angle`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "63e15583",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Help on function rx in module braket.experimental.autoqasm.instructions.gates:\n",
+      "\n",
+      "rx(target: Union[int, oqpy.classical_types._ClassicalVar, oqpy.base.OQPyExpression, str, oqpy.quantum_types.Qubit], angle: float) -> None\n",
+      "    X-axis rotation gate.\n",
+      "    \n",
+      "    Args:\n",
+      "        target (QubitIdentifierType): Target qubit.\n",
+      "        angle (float): Rotation angle in radians.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "help(rx)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1189626",
+   "metadata": {},
+   "source": [
+    "The pulse operations of the gate calibration is composed in the body of a function decorated by `@aq.gate_calibration`. To specify fixed values for the parameters `target` and `angle`, we set the values through keyword arguments of the decorator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "id": "64ebd3a2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wf = DragGaussianWaveform(\n",
+    "    24e-9, 2.547965400864e-9, 2.370235498840002e-10, 0.293350447987059, False, \"wf_drag_gaussian_1\"\n",
+    ")\n",
+    "q0_rf_frame = device.frames['q0_rf_frame']\n",
+    "\n",
+    "@aq.gate_calibration(implements=rx, target=\"$0\", angle=np.pi/2)\n",
+    "def my_defcal():\n",
+    "    pulse.barrier(\"$0\")\n",
+    "    pulse.shift_frequency(q0_rf_frame, -321047.14178613486)\n",
+    "    pulse.play(q0_rf_frame, wf)\n",
+    "    pulse.shift_frequency(q0_rf_frame, 321047.14178613486)\n",
+    "    pulse.barrier(\"$0\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9300f0f8",
+   "metadata": {},
+   "source": [
+    "After the gate calibration is defined, it is ready to be attach to a quantum program. Say, we want to attach `my_defcal` to the main quantum program `my_program`."
```

Comment:
The next box doesn't attach the calibration to the program, so you should reword this to only introduce the next code block (with context of why you're introducing it)